types.d.ts

```diff
@@ -40,3 +40,9 @@ export interface ApiData {
   date: string;
   metrics: { [key: string]: MetricsData };
 }
+
+export interface UserType { // Define an interface for user schema properties
```

Comment:
possibly put this in the userModel since it is only being used

Reply:
updated!